src/gf_ring.h

```diff
@@ -534,24 +559,29 @@ bool RingModN<T>::is_prime_root(T nb)
     return ok;
 }
 
-/*
- * Find primitive root of finite field. A number x is a primitive root if its
- *  order y = q - 1, i.e. x^i != 1 for every i < q-1
+/** Find primitive root of finite field.
+ *
+ * A number `x` is a primitive root if its order \f$y = q - 1\f$, i.e.
+ * \f$x^{i}\neq 1\qquad (i=1, 2, 3, \dots , q-1)\f$
```

Comment:
Same above, should it be .., \dots , q-2)\f$?

Reply:
Done